### DISCUSSION TITLE
Library / Album view window only displays the last track

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1344] [Fixed Issues]
Running rAudio 20230603

Using the Library / Album function, the display initially shows only the last track of the album.

Initial view
<img src="https://github.com/rern/rAudio/assets/76575081/64270d8c-77c3-4ca9-8659-e3a9650eb06b)" width=300>

View after scrolling to the top of window
<img src="https://github.com/rern/rAudio/assets/76575081/424abadb-2f42-4294-ada3-a36c572acbaf)" width=300>

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

Jim

**Comment by ceejayemm** [comment]
Seems to be fixed in 20230609.

Chris

**Reply by Sagnasty** [reply]
I have the same display behavior with 20230609 (iPhone, Firefox, and local browser)

Only the Library Album views are incorrect, Album artist and artist are correct.

Jim

**Reply by Sagnasty** [reply]
Additional info:  the reported behavior does not always occur.   Some album views are displayed correctly while the majority are not.

**Comment by Editpo** [comment]
I am getting this problem as well. Not fixed for me in 0609

**Comment by kw-sudo** [comment]
For me, no song is displayed at all, I always have to scroll down completely. Currently use i20230521 + pacman -Syu.

RasPi 3B+

**Reply by Sagnasty** [reply]
I have seen this view as well.

**Comment by ceejayemm** [comment]
I thought this was fixed earlier, again it seems to be one of the 'new' on/off problems we are seeing with the recent releases.  For me it works correctly for some albums but for others the album title is displayed but then I am taken to the end of the track list so at first glance it seems the tracks are missing but if you scroll up they are in fact all present and correct. I have no idea at the moment why this happens to some albums and not others..

Chris

**Reply by kw-sudo** [reply]
After spending all afternoon yesterday dealing with i20230521 and ultimately still getting errors, I went back to i20230203 + pacman -Syu this morning.

My RasPi is only active in LAN (no external access) and r/Audio does exactly what I need (NAS>RasPi>Amp (LAN/USB)).

I will stay tuned... ;-)
Oliver

**Comment by Sagnasty** [comment]
Fixed in 20230615 - thanks Rern
